packages/editor/src/components/editor/canvas/Canvas.tsx

```diff
@@ -34,8 +42,22 @@ export const ComponentBlock = ({ component, config, preId, ...props }: Component
   </DropZone>
 );
 
-export const EmtpyBlock = ({ id, preId, forLayout }: { id: string; preId: string; forLayout?: boolean }) => (
+export const EmtpyBlock = ({
+  id,
+  preId,
+  forLayout,
+  dragHint
+}: {
+  id: string;
+  preId: string;
+  forLayout?: boolean;
+  dragHint?: { display: boolean; mode: 'row' | 'column'; message: string };
+}) => (
```

Comment:
extract to own prop type, to prevent this format

Reply:
the mode and the message type could also be extracted from the EmptyDetail component.